<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="JarvisLaptop" failures="1" tests="2" name="Sanity" time="11.644" errors="0" timestamp="2023-03-08T07:47:20 EET">
  <testcase classname="com.Restaurant.Testcases.VerifyvaliduserLoginTest" name="VerifyUserLogin" time="1.694">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.Restaurant.Testcases.VerifyvaliduserLoginTest.VerifyUserLogin(VerifyvaliduserLoginTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- VerifyUserLogin -->
  <testcase classname="com.Restaurant.Testcases.VerifyvaliduserLoginTest2" name="VerifyUserLogin" time="0.888"/>
  <testcase classname="com.Restaurant.BaseClass.BaseClass" name="@BeforeSuite loadConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite loadConfig -->
  <testcase classname="com.Restaurant.BaseClass.BaseClass" name="@AfterSuite Closing" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite Closing -->
</testsuite> <!-- Sanity -->
